### PR TITLE
[IMP] hr: adding distance home-work unit field

### PR DESCRIPTION
**Pull request by dhka-odoo**
adding new selection field distance home-work unit for changing between miles and km 
added a new field for computation of miles into km changing field name from km_home_work to distance_home_work 
km_home_work will be used for calculation in km and distance_home_work will be used for input

Task-3888505

### CODE REVIEW

**Review by robcoekaerts — COMMENTED** [review]
Some small comments

File: addons/hr/models/hr_employee.py [hunk 1]
```diff
@@ -115,7 +115,12 @@ def _lang_get(self):
     study_school = fields.Char("School", groups="hr.group_hr_user", tracking=True)
     emergency_contact = fields.Char("Contact Name", groups="hr.group_hr_user", tracking=True)
     emergency_phone = fields.Char("Contact Phone", groups="hr.group_hr_user", tracking=True)
-    km_home_work = fields.Integer(string="Home-Work Distance", groups="hr.group_hr_user", tracking=True)
+    distance_home_work = fields.Integer(string="Home-Work Distance", groups="hr.group_hr_user", tracking=True)
+    km_home_work = fields.Integer(string="Home-Work Distance in Km", groups="hr.group_hr_user", compute="_compute_km_home_work", store=True)
+    distance_home_work_unit = fields.Selection([
+        ('kilometers', 'Km'),
```
Comment (robcoekaerts) [inline addons/hr/models/hr_employee.py:121]: the unit should not be capitalized, so it should be `'km'`

File: addons/hr/models/hr_employee.py [hunk 2]
```diff
@@ -115,7 +115,12 @@ def _lang_get(self):
     study_school = fields.Char("School", groups="hr.group_hr_user", tracking=True)
     emergency_contact = fields.Char("Contact Name", groups="hr.group_hr_user", tracking=True)
     emergency_phone = fields.Char("Contact Phone", groups="hr.group_hr_user", tracking=True)
-    km_home_work = fields.Integer(string="Home-Work Distance", groups="hr.group_hr_user", tracking=True)
+    distance_home_work = fields.Integer(string="Home-Work Distance", groups="hr.group_hr_user", tracking=True)
+    km_home_work = fields.Integer(string="Home-Work Distance in Km", groups="hr.group_hr_user", compute="_compute_km_home_work", store=True)
+    distance_home_work_unit = fields.Selection([
+        ('kilometers', 'km'),
+        ('miles', 'mi'),
+    ], 'Home-Work Distance unit', tracking=True, groups="hr.group_hr_user", default='kilometers')
```
Comment (KangOl) [inline addons/hr/models/hr_employee.py:123]:
```suggestion
    ], 'Home-Work Distance unit', tracking=True, groups="hr.group_hr_user", default='kilometers', required=True)
```

File: addons/hr/models/hr_employee.py [hunk 3]
```diff
@@ -115,7 +115,12 @@ def _lang_get(self):
     study_school = fields.Char("School", groups="hr.group_hr_user", tracking=True)
     emergency_contact = fields.Char("Contact Name", groups="hr.group_hr_user", tracking=True)
     emergency_phone = fields.Char("Contact Phone", groups="hr.group_hr_user", tracking=True)
-    km_home_work = fields.Integer(string="Home-Work Distance", groups="hr.group_hr_user", tracking=True)
+    distance_home_work = fields.Integer(string="Home-Work Distance", groups="hr.group_hr_user", tracking=True)
+    km_home_work = fields.Integer(string="Home-Work Distance in Km", groups="hr.group_hr_user", compute="_compute_km_home_work", store=True)
+    distance_home_work_unit = fields.Selection([
```
Comment (KangOl) [inline addons/hr/models/hr_employee.py:120]: Actually, why not use uom?

**Comment by tivisse** [comment]
@robodoo r+

File: addons/hr/models/hr_employee.py (hunk 3 above)
Comment (tivisse) [reply addons/hr/models/hr_employee.py:120]: This was a functional decision :man_shrugging:

**Comment by dhka-odoo** [comment]
> @robodoo r+

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**